src/mixins/update-user-mixin.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { updateLdUser } from '@/plugins'
+import AuthServices from '@/services/auth.services'
+
+@Component({})
+export class UpdateUserMixin extends Vue {
```

Comment:
This code is simply ported from Filings UI.

I put it in a mixin because I didn't like the Signin page calling an auth service directly.